src/api/operations.ts

```diff
@@ -27,10 +25,12 @@ import {
   setLineCap,
   setGraphicsState,
   setDashPattern,
+  appendBezierCurve,
 } from 'src/api/operators';
 import { Rotation, toRadians } from 'src/api/rotations';
 import { svgPathToOperators } from 'src/api/svgPath';
 import { PDFHexString, PDFName, PDFNumber, PDFOperator } from 'src/core';
+import {asNumber} from "./objects";
```

Comment:
Can we revert this import to what's in `master`?
```suggestion
import { asNumber } from 'src/api/objects';
```